README.md

```diff
@@ -29,6 +29,15 @@ npm run serve
 TV_IP=10.0.1.1 npm run test-tv
 ```
 
+## Testing Agate components

```

Comment:
When it is not agate, there is no explanation as to which component is running.
How about mentioning that existing commands run on sandstone?

Reply:
Currently, there is no command combination to test the agate component on TV.
It seems better to use the --target and --library options than creating a new command.

Please remove commands containing -agate & -tv such as test-agate and test-tv and change the --libarary option to include sandstone or agate such as `npm run test -- --target=PC --library=agate`

And please change the environment variable to operate by setting it to a string, such as ENACT_THEME="agate" or ENACT_THEME="sandstone", considering that tests for components other than agate are added later.

Reply:
> Currently, there is no command combination to test the agate component on TV. 

Since agate is a library targeted for automotive, do we need to test it on TV?

> It seems better to use the --target and --theme options than creating a new command.

For commands running in NodeJS environment (npm run test) this is a good suggestion, but `enact serve` should have 2 commands since in browser environemnt we can't read command line arguments like `--theme`.

> And please change the environment variable to operate by setting it to a string, such as ENACT_THEME=agate or ENACT_THEME=sandstone, considering that tests for themes other than agate may be added.

Ok, but I have to keep `REACT_APP` prefix, otherwise React doesn't know how to read env variables.